### ISSUE TITLE
Accessibility Modal Update Requests

### ISSUE THREAD

**Issue by melissahermes99** [hackforla/311-data#1458]
### Overview
After some review of V2 on the dev site, there are a few updates we would like to see on the site header (we need to add accessibility modal).

### Action Items
- [x] ~Remove the privacy tap from the header. After some discussion amongst the design team, we felt that housing the Privacy tab in the header seemed repetitive because it is also located in the footer. For now, we would like to remove it, but we can also consider testing it with users to see what is most intuivite.~
- [ ] We currently do not have an accessibility button on Version 2, we feel that a good location for that would be on the very right hand corner of the header.

### Resources/Instructions
- Link to Epic with figma resources: https://github.com/hackforla/311-data/issues/1625

**Comment by funbunch** [comment]
Hi @melissahermes99 Can you direct to me where to get that new accessibility button on the Figma?

**Comment by funbunch** [comment]
@melissahermes99 @edwinjue  Not sure of Michaal's? username to tag.
Status update: I was able to find and add the button. But am blocked on completing this PR until I find out what functionality the button actually does. And if I add that here or if that should be completed on another ticket.

**Comment by melissahermes99** [comment]
@funbunch 
![image](https://user-images.githubusercontent.com/97048386/220762160-62ad097d-50ea-4469-8e83-25a0373d2ccc.png) Is this what you were looking for? This is the function of the accessibility icon. It should work like any of the other navigation tabs however, It is more of a dropdown than a page so that it is accessible throughout the site. It is located on the Version 2 Iterations page in Figma and the actual drop down is located in the components section. https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=1009%3A1004

**Comment by funbunch** [comment]
@melissahermes99 Yes thanks!

**Comment by funbunch** [comment]
I found this again. I can keep working on it if still good. @ryanfchase @edwinjue

**Comment by edwinjue** [comment]
Hi @funbunch. How are you? Apologies for the belated response. Yes, I believe this is something we still need to implement for the front end. Could you please take a look when you have a chance? Please feel free to ping me on slack to discuss further if you need.

**Comment by annaseulgi** [comment]
Hi all, will the Privacy text link be removed from the nav bar? If so, please let design know so we can update the component on our end in the Figma. Thanks!

**Comment by edwinjue** [comment]
Hi @annaseulgi @funbunch which is the correct design for the Accessible Menu dropdown? Thanks
Design 1:
![image](https://github.com/hackforla/311-data/assets/1448719/9b17de6c-a4fa-4c4c-9c27-da79044e81ea)

Design 2:
![image](https://github.com/hackforla/311-data/assets/1448719/db22a97c-d916-4612-b665-c68dd2b8ce58)

**Comment by funbunch** [comment]
Hey @edwinjue. I had spoken with @annaseulgi about it and she wanted to go with the top light one.

But feel free to QA this for design @annaseulgi!

**Comment by edwinjue** [comment]
Thanks @funbunch. Could you or @annaseulgi provide a link to this design so I can check it out? Thanks

**Comment by annaseulgi** [comment]
@edwinjue the design is included in the Figma page titled "⭐⭐⭐ V1.2 HAND-OFF". All designs will be located in the same Figma. (I know previously Graem used a different Figma file for his FAQ hand off but moving forward, the design team will centralize all designs in the designed 311 Figma.)

For your convenience: https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?type=design&node-id=4081%3A9245&mode=design&t=rdv9kaSWaR4jm2PU-1

Thanks!

**Comment by annaseulgi** [comment]
@edwinjue @ryanfchase Would y'all mind creating a new issue for this? Looks like the title is called Header Update Requests but we're also discussing the accessibility modal.

**Comment by funbunch** [comment]
@edwinjue Merged in changes for Node 18 and updated Header file to work with the new MUI.

**Comment by ryanfchase** [comment]
I'm going to update the title (rather than make a new issue for this) since design lead mentioned title is no longer accurate. Keeping this issue because there is already a PR on this ticket.

**Comment by ryanfchase** [comment]
@funbunch left feedback on the PR, ty!
